springfox-schema/src/main/java/springfox/documentation/schema/DefaultModelProvider.java

```diff
@@ -109,10 +109,20 @@ private Model modelBuilder(ResolvedType propertiesHost,
         .description("")
         .baseModel("")
         .discriminator("")
-        .subTypes(new ArrayList<String>());
+        .subTypes(new ArrayList<String>())
+        .xml(buildXml());
     return schemaPluginsManager.model(modelContext);
   }
 
+  private Xml buildXml() {
+    return new Xml()
+        .name("")
+        .attribute(false)
+        .namespace("")
+        .prefix("")
+        .wrapped(false);
```

Comment:
This doesnt seem right. We shouldn't always initialize the xml. this has resulted in all the contracts having the xml element in them.

Also we need a way to grab the right xml namespace/prefix/name from the `XmlSchema` annotation.